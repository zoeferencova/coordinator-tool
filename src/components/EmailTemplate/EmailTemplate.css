.template-item {
    background-color: lightgray;
    height: 60px;
    line-height: 60px;
    margin: 0;
    border-radius: 10px;
}

.template-body {
    white-space: pre-line;
}

.template-contents p {
    text-align: left;
}

.template-item:hover {
    cursor: pointer;
}