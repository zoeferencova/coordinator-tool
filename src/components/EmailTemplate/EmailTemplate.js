import React from 'react';
import styles from './EmailTemplate.module.css'

const EmailTemplate = ({ id, selectTemplate, template_name, current }) => (
    <div>
        <div className={`${styles.templateItem} ${current === id ? styles.current : ''}`} onClick={() => selectTemplate(id)}>
            {template_name}
        </div>
    </div>
)

export default EmailTemplate;