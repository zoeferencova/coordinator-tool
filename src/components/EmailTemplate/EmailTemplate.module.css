.templateItem {
    padding: 10px;
    text-align: left;
    font-size: 13px;
}

.templateItem:hover {
    cursor: pointer;
}