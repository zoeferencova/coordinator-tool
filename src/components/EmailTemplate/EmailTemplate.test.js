import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import EmailTemplate from './EmailTemplate'

describe(`EmailTemplate component`, () => {
  it('renders a .EmailTemplate by default', () => {
    const wrapper = shallow(<EmailTemplate />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})