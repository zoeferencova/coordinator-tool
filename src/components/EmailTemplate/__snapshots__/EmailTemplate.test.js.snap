// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailTemplate component renders a .EmailTemplate by default 1`] = `
<div>
  <div
    className="templateItem current"
    onClick={[Function]}
  />
</div>
`;
